### PR TITLE
Recalculate label counts periodically

### PR DESCRIPTION
**Pull request by barryam3**
Wanted you to take a look at this before bringing this change into master. The nature of cleaning things up on a timer makes it hard to write automated tests for. I tested the count query in phpMyAdmin and it seems good. The dev server also starts fine.

### CODE REVIEW

File: database/dbops.js [hunk 1]
```diff
@@ -86,6 +89,40 @@ async function getNextLevelNum(userID) {
   return levels[0].levelsCount + 1;
 }
 
+// used by fixLabelCounts below
+const durHMS = secToHMS(MAX_LEVEL_TIME_SEC);
```
Comment (a-newman) [inline database/dbops.js]: I'm guessing that this turns seconds into a SQL-legible time string?

File: database/dbops.js [hunk 2]
```diff
@@ -13,6 +15,7 @@ const VID_TYPES = {
 }
 
 const N_LEVELS_PER_NEW_LIFE = 50;
+const MAX_LEVEL_TIME_SEC = 1 * 60 * 60; // 1 hour
```
Comment (a-newman) [inline database/dbops.js]: Seem reasonable to me; we might want to drop this to half an hour in the future if we realize that stuff gets out of sync quickly.

File: bin/www [hunk 3]
```diff
@@ -80,6 +81,8 @@ function onListening() {
  * Listen on provided port, on all network interfaces.
  */
 initDB().then(() => {
+  fixLabelCounts();
+  setInterval(fixLabelCounts, MAX_LEVEL_TIME_SEC * 1000);
```
Comment (a-newman) [inline bin/www]: I wonder if we should not run this automatically to start off; maybe we can run it manually a few times first to make sure it doesn't do something unexpected?

File: test/test.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 const request = require('supertest');
-const app = require('../app');
 const debug = require('debug')('memento:server');
```
Comment (a-newman) [inline test/test.js:2]: Even though we can't test that this is actually running at a given interval, we should be able to test the functions (fixLabelCounts, for instance). I think a simple test would be warranted, to make sure the query is right. There's already some code inside this file to wipe the db before/after a test; see "Test video prioritization".

**Review by a-newman — COMMENTED** [review]
Looks great, thanks Barry!

File: database/dbops.js (hunk 2 above)
Comment (barryam3) [reply database/dbops.js]: Possibly. I wanted to be sure I wasn't going to give anyone with a slow connection trouble. But an hour might be overkill.

File: bin/www (hunk 3 above)
Comment (barryam3) [reply bin/www]: Hmm, I would think if there were problems they would show up on the first run. Now that there are tests passing I feel more confident about it. But if you want to try running it before merging go ahead.

Comment (a-newman) [reply bin/www]: ok, sounds good to me. We can try running another 100 rounds on mturk today

**Pull request merged**